apps/client/src/app/app-routing.module.ts

```diff
@@ -76,6 +62,20 @@ const routes: Routes = [
     loadChildren: () =>
       import('./pages/tools/tools-page.module').then((m) => m.ToolsPageModule)
   },
+  {
+    path: 'tools/analysis',
```

Comment:
I think that these sub-paths can be declared here as tools children routes: https://github.com/ghostfolio/ghostfolio/blob/main/apps/client/src/app/pages/tools/tools-page-routing.module.ts#L8

What do you think? @dtslvr

Reply:
Not sure how to add `AnalysisPageModule` [here](https://github.com/ghostfolio/ghostfolio/blob/main/apps/client/src/app/pages/tools/tools-page-routing.module.ts#L8) 🤔